cmd/devstream/develop.go

```diff
@@ -32,7 +32,7 @@ func developCMDFunc(cmd *cobra.Command, args []string) {
 	}
 }
 
-func validateArgs(args []string) error {
+func validateDevelopArgs(args []string) error {
 	// "create-plugin"/ maybe it will be "delete-plugin"/"rename-plugin" in future.
 	if len(args) != 1 {
 		return fmt.Errorf("got illegal args count (expect 1, got %d). "+
```

Comment:
Let's refactor this error message.

It should not be a full sentence. "illegal args count", "missing arg(s)" is better, remove the verb "got".

We should not add another full sentence "See `help` command for more info".

It will look like "got illegal args count (expect 1, got 0). See `help` command for more info", it's not a full sentence as a log line.

Reply:
return fmt.Errorf("illegal args count (expect 1, got %d)", len(args))